cypress/e2e/workspace.spec.js

```diff
@@ -323,3 +325,21 @@ function testButton(button, tag, content) {
 	cy.getMenuEntry(button).click()
 	cy.getMenuEntry(button).should('not.have.class', 'is-active')
 }
+
+/**
+ *
+ * @param {string} button Name of the button to click.
+ * @param {string} tag Html tag expected to be toggled.
+ * @param {string} content Content expected in the element.
+ */
+function testButtonUnselected(button, tag) {
+	cy.getMenuEntry(button).click().should('have.class', 'is-active')
+	cy.getContent().type('Format me{selectall}')
+		.find(`${tag}`)
+		.should('contain', 'Format me').type('{del}')
+	cy.getMenuEntry(button).click().should('have.class', 'is-active').click().should('not.have.class', 'is-active')
```

Comment:
Cypress eslint extension is complaining about these lines because you are chaining off commands.
What does it mean by that:
```javascript
cy.getContent().type('Format me{selectall}').find(...)
```
`.type()` is a command. (So is `.click()`).

The problem is rooted in how cypress works. For queries like `find()` it will look for the element. If it does not exist yet it will retry until the element exists or it times out.
If the `find()` command is chained from the `type()` command there's no safe way for cypress to retry without calling the `type()` command multiple times.

So whenever you have a command that actually changes something you need to end the chain:
```suggestion
	cy.getContent().type('Format me{selectall}')
	cy.getContent().find(`${tag}`)
		.should('contain', 'Format me').type('{del}')
	cy.getMenuEntry(button).click()
	cy.getMenuEntry(button).should('have.class', 'is-active').click()
	cy.getMenuEntry(button).should('not.have.class', 'is-active')
```

This is only checked since a short time - so you will still find old documentation and even code that does not follow this rule.